app/CocoaPods/CPPodfileSyntaxChecker.h

```diff
@@ -0,0 +1,21 @@
+//
+//  CPPodfileSyntaxChecker.h
+//  CocoaPods
+//
+//  Created by Orta on 11/27/15.
+//  Copyright © 2015 CocoaPods. All rights reserved.
+//
+
+#import <Cocoa/Cocoa.h>
+
+@class MGSFragariaView, CPPodfileEditorViewController;
+
+/// Handles keeping on top of the syntax for the Podfile
+/// at runtime. Note: Comes with a side-effect of setting the
+/// plugins array on the
```

Comment:
I think there's a different space for something that does the reflection needed to pull off the Podfile info design, this doesn't have to be in the same class as the thing that acts as our syntax linter however.